frontend/src/plugins/issue/logic/initialize/index.ts

```diff
@@ -42,24 +54,24 @@ export function useInitializeIssue(issueSlug: Ref<string>) {
   watch(
     [issueSlug, create],
     async ([issueSlug, create]) => {
-      issue.value = undefined;
-
       try {
         if (create) {
+          issueCreate.value = undefined;
+
           await prepareDatabaseListForIssueCreation(route.query);
 
-          issue.value = await buildNewIssue({ template, route });
+          issueCreate.value = await buildNewIssue({ template, route });
           if (
-            issue.value.assigneeId === UNKNOWN_ID ||
-            issue.value.assigneeId === SYSTEM_BOT_ID
+            issueCreate.value.assigneeId === UNKNOWN_ID ||
+            issueCreate.value.assigneeId === SYSTEM_BOT_ID
           ) {
             // Try to find a default assignee of the first task automatically.
-            await tryGetDefaultAssignee(issue.value);
+            await tryGetDefaultAssignee(issueCreate.value);
           }
         } else {
           const id = idFromSlug(issueSlug);
-          const fetchedIssue = await issueStore.fetchIssueById(id);
-          issue.value = fetchedIssue;
+          await new Promise((resolve) => setTimeout(resolve, 15000));
```

Comment:
Maybe add a comment here about why sleep 15s.

Reply:
sry for that. updated.